articles/deep-neural-system-matlab/index.md

```diff
@@ -0,0 +1,214 @@
+### How to create a deep neural network using MATLAB.
+### Introduction.
+Neural networks form the base of deep learning and a subfield of machine learning where algorithms are inspired by the structure of the human brain. It takes in data train themselves to recognize the pattern in this data and then predict the output for a new set of similar data.
```

Comment:
Consider changing "Neural networks form the base of deep learning and a subfield of machine learning..." to "Neural networks form the base of deep learning, a subfield of machine learning..." . "It takes in data train themselves to recognize the pattern in this data..." to "It takes in data, train itself to recognize the pattern in this data..."